examples/custom-properties/http.js

```diff
@@ -41,7 +43,12 @@ app.use(async ({ logger, context, next }) => {
 
 (async () => {
   // Start your app
-  await app.start(process.env.PORT || 3000);
-
+  try {
+    await app.init();
```

Comment:
Example error pattern:

```
> bolt-js-custom-properties-app@1.0.0 start /Users/ksera/github/bolt-js/examples/custom-properties
> node http.js

AppInitializationError: Apps used in a single workspace can be initialized with a token. Apps used in many workspaces should be initialized with oauth installer options or authorize.

Since you have not provided a token or authorize, you might be missing one or more required oauth installer options. See https://slack.dev/bolt-js/concepts#authenticating-oauth for these required fields.

    at App.initAuthorizeIfNoTokenIsGiven (/Users/ksera/github/bolt-js/dist/App.js:660:19)
    at App.init (/Users/ksera/github/bolt-js/dist/App.js:180:47)
    at /Users/ksera/github/bolt-js/examples/custom-properties/http.js:46:15
    at Object.<anonymous> (/Users/ksera/github/bolt-js/examples/custom-properties/http.js:53:3)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)
    at internal/main/run_main_module.js:18:47 {
  code: 'slack_bolt_app_initialization_error'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 255
npm ERR! bolt-js-custom-properties-app@1.0.0 start: `node http.js`
npm ERR! Exit status 255
```